Nightly is failing in phpstan

The [nightly](https://drone.owncloud.com/owncloud/guests/3077/2/4) is failing with following error

```bash
 -- ----------------------------------------------------------------------------- 
     Error                                                                        
 -- ----------------------------------------------------------------------------- 
     Ignored error pattern #Method                                                
     Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\)  
     invoked with 2 parameters, 1 required.# was not matched in reported          
     errors.                                                                      
 -- ----------------------------------------------------------------------------- 


 [ERROR] Found 1 error                                                          

make: *** [Makefile:146: test-php-phpstan] Error 1
```

Related core issue: https://github.com/owncloud/core/issues/39630

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -6,7 +6,6 @@ parameters:
     - %currentWorkingDirectory%/appinfo/Migrations/*.php
     - %currentWorkingDirectory%/appinfo/routes.php
   ignoreErrors:
-    - '#Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\) invoked with 2 parameters, 1 required.#'
     -
       message: '#Property OCA\\Guests\\Controller\\RegisterController::\$groupManager is never read, only written.#'
       path: lib/Controller/RegisterController.php

